VideoLocker/res/values/strings.xml

```diff
@@ -450,6 +448,8 @@
     <string name="title_lecture_complete">You\'ve reached the end of this video.</string>
     <!-- Alert dialog prompt at end of video. -->
     <string name="msg_lecture_complete">Would you like to watch the next video?</string>
+    <!-- Accessible content description of cancel button to cancel video download -->
+    <string name="cancel">Cancel</string>
```

Comment:
Using framework strings would make our coverage inconsistent though - in unsupported locales there will be some strings displaying in the language (by the framework) while most of the app shows up in English.
> On 11-Sep-2015, at 8:33 pm, Brian Guertin <notifications@github.com> wrote:
> 
> In VideoLocker/res/values/strings.xml <https://github.com/edx/edx-app-android/pull/354#discussion_r39283428>:
> 
> > @@ -450,6 +448,8 @@
> >      <string name="title_lecture_complete">You\'ve reached the end of this video.</string>
> >      <!-- Alert dialog prompt at end of video. -->
> >      <string name="msg_lecture_complete">Would you like to watch the next video?</string>
> > +    <!-- Accessible content description of cancel button to cancel video download -->
> > +    <string name="cancel">Cancel</string>
> You can also use the O.S. provided cancel string, then it will be translated for us: android.R.id.cancel
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/354/files#r39283428>.
>